public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
+          fields: {
+            '*': {
+              fragment_size: 0,
```

Comment:
What happens if `fragment_size` is 0 ? It is the number of characters to highlight, right https://www.elastic.co/guide/en/elasticsearch/reference/2.4/search-request-highlighting.html#_highlighted_fragments ?